### ISSUE TITLE
Building docker image throws error

### ISSUE THREAD

**Issue by pasa13142** [jina-ai/clip-as-service#926]
Hello, 

Do you know how can I get "jinaai/clip-server:latest-onnx"'s dockerfile? 

Building&Running from repo with PIP_TAG=onnx maybe 3 months ago works well but vector values are different with latest-onnx. 

Also, I tried today

```
git clone https://github.com/jina-ai/clip-as-service.git
docker build . -f Dockerfiles/server.Dockerfile  --build-arg GROUP_ID=$(id -g ${USER}) --build-arg USER_ID=$(id -u ${USER}) -t jinaai/clip-server
```

As I said it was working but it is not working anymore. It throws:
`manifest for nvidia/cuda:11.6.0-devel-ubuntu20.04 not found: manifest unknown: manifest unknown`

In short, Is there any chance can we have "jinaai/clip-server:latest-onnx"'s dockerfile?

**Comment by kindziora** [comment]
Problem seems the cuda image version. 
change it to `ARG CUDA_VERSION=11.6.2` that did work for me... kind regards

**Comment by pasa13142** [comment]
Thanks for the answer! I reported it also to fix for new users.

docker run -p 51009:51000 -v $HOME/.cache:/home/cas/.cache --gpus all jinaai/clip-server:master-onnx onnx-flow.yml

the encodes extracted from above server and when you remove --gpus all parameter (CPU) encodes are different. How can I avoid it?

**Comment by kindziora** [comment]
Hi pasa, i'am just guessing because i just started using this project. I think there could be several reasons for that:

-  maybe the order of your input data is changing
-  maybe cpu mode does some different preprocessing on your input(fp16 vs fp32).  

but i wonder if you even get the same results on 2 different instances both running with gpu.

kind regards

**Comment by ZiniuYu** [comment]
Yes the base image from Nvidia with cuda == 11.6.0 was deleted, please use 11.6.2 instead

> Problem seems the cuda image version.
> change it to `ARG CUDA_VERSION=11.6.2` that did work for me... kind regards

**Comment by ZiniuYu** [comment]
> Thanks for the answer! I reported it also to fix for new users.
> 
> docker run -p 51009:51000 -v $HOME/.cache:/home/cas/.cache --gpus all jinaai/clip-server:master-onnx onnx-flow.yml
> 
> the encodes extracted from above server and when you remove --gpus all parameter (CPU) encodes are different. How can I avoid it?

What do you mean the encodes are different? Can you show some examples?
The fp16 will lose some precisions by natural, some slight differences would be expected
